great_expectations/experimental/metric_repository/metrics.py

```diff
@@ -56,17 +68,98 @@ def update_forward_refs(cls):
             Batch=Batch,
         )
 
+    @property
+    def value_type(self) -> str:
+        type_ = self.__orig_class__.__args__[0]
+        string_rep = str(type_)
+        if string_rep.startswith("<class"):
+            return type_.__name__
+        else:
+            return string_rep
+
+    @property
+    def metric_type(self) -> str:
+        return self.__class__.__name__
```

Comment:
These properties are to retrieve the value_type and metric_type for serializing in the payload so that on the backend we can use them to find the right metric type for deserialization.